fhir-install/src/main/assembly/distribution.xml

```diff
@@ -106,5 +106,14 @@
                 <include>org.postgresql:postgresql</include>
             </includes>
         </dependencySet>
+        <dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <outputDirectory>tools</outputDirectory>
+            <directoryMode>0755</directoryMode>
+            <fileMode>0755</fileMode>
+            <includes>
+                <include>${project.groupId}:fhir-persistence-schema:jar:cli</include>
+            </includes>
+        </dependencySet>
```

Comment:
May want to simplify and add the other clis mentioned here, and we. can. have one deliverable to upload? 
https://github.com/IBM/FHIR/wiki/Release-Guide#release

Reply:
i think this is the only one that we need at runtime; thus to me it makes sense to keep the others outside of the distribution (and users can continue to get them independent of the distribution zip)